tests/acceptance/26_cf-net/serial/cf-net_connect.cf

```diff
@@ -0,0 +1,52 @@
+body contain myshell
+{
+    !windows::
+        useshell => "useshell";
+    windows::
+        useshell => "powershell";
+}
+
+body common control
+{
+    inputs => {
+        "../../default.cf.sub",
+        "../../run_with_server.cf.sub"
+    };
+    bundlesequence => { default("$(this.promise_filename)") };
+}
+
+bundle agent init
+{
+    methods:
+        "any" usebundle => dcs_fini("$(G.testdir)/cf-net_connect.txt");
+
+        "any" usebundle => generate_key;
+        "any" usebundle => trust_key;
+
+        "any" usebundle => start_server("$(this.promise_dirname)/cf-net_connect.cf.srv");
```

Comment:
I presume the same server file will be used for more tests, so you should name it independent of the cf-net test name, based on the server functionality, for example "serverd_admit_localhost.srv".